Header commandHeader
int16 planeID
bool sim
int16 commandID
int16 param
float64 latitude
float64 longitude
float64 altitude
